OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -184,7 +184,7 @@ public void UnmountAll()
 		public void TrimExcess()
 		{
 			mountedPackages.TrimExcess();
-			explicitMounts.TrimExcess();
+			explicitMounts?.TrimExcess();
```

Comment:
`explicitMounts` is never null.

Reply:
it was null when I was running the extraction rule for mode content. See the commit `Fix explicit mounts not being optional`

Reply:
`explicitMounts` is never null by construction - are you sure it was this field? Either that of there's some reflection going on that I cannot trace.

Reply:
interesting, it no longer crashes